app/api/items/testdata/restoredb.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Yeah, I know about it.

Reply:
But it depends on mysql, which not installed on my pc.
And I dont want to change the config of docker-compose.
The password is in the projects files on github already.
I dont think it will be used for production env...
So I use the mysql command in the docker instead of install anything.

Reply:
nothing in what was in place required to have mysql installed